src/entry.ts

```diff
@@ -214,15 +214,16 @@ class Entry {
   private formatHttpRequest() {
     const rawReq = this.metadata.httpRequest;
     if (rawReq) {
-      // Handle raw http requests.
+      // Handle raw http request.
       if (
         'statusCode' in rawReq ||
         'headers' in rawReq ||
         'method' in rawReq ||
         'url' in rawReq
       )
         this.metadata.httpRequest = makeHttpRequestData(rawReq);
-      // Infer trace & span if not user specified already.
+      // Infer trace & span if not user specified already
+      // TODO(nicolezhu): also infer from trace header if OpenCensus is configured
```

Comment:
can you open an issue on this in the repo?

Reply:
opened!